### ISSUE TITLE
Redo Please

### ISSUE THREAD

**Issue by PIRANY1** [pfpz/SAK-MultiTool#1]
Your script is kinda ok, but why does it  need to be in the sys directory??. Also why dont you integrate the VBS scripts in the main batch file and then execute them from there. Else the Script is great. Why do you have system requierments?????? batch runs on a fucking potatoe please omg

**Comment by pfpz** [comment]
Thank you for your feedback I greatly appreciate it! I can definitely make
iterations to the script to allow it to be used in any directory, but if
you could, can you elaborate more on what you mean by intergratung the VBS
scripts? Do you mean to do it the same way as I did inside the program
using powershell? Please let me know, thank you so much!

**Comment by PIRANY1** [comment]
Yeah i can definitely do that. I mean it so that the Script creates the VBS File in the Temp directory and executes it from there

**Comment by pfpz** [comment]
I have done that in a previous version but I saw it to be alot slower, I
can see what I can do to definitely make it more efficient though! I have
been working on a more efficient version of the script and looking into
different methods of coding to make it more interesting so I really
appreciate your feedback!

**Comment by PIRANY1** [comment]
There is tbh not really any use case for this script but il do it anyways. but i would highly recommend that you learn a better language like c++ (https://www.learncpp.com) its just better instead of wasting time on batch

**Comment by PIRANY1** [comment]
If you want to stay with batch you can use my script as inspiration if you want www.github.com/pirany1/dataspammer

**Comment by pfpz** [comment]
I would love to learn c++ but I’m just a beginner so I have no idea where
to start or anything. I have began learning html, css and java script and I
enjoy that (https://imhereforyou.online & https://digitalwarlibrary.online),
but I definitely have some years before I get good at this. I mainly
started coding back in 2020 during covid for fun and made some little batch
scripts and then gave up, but more recently I got back into it and I really
wanna pursue it as a career because its extremely interesting!

**Comment by pfpz** [comment]
Thank you! I will definitely check it out

**Comment by PIRANY1** [comment]
Hey i reworked a big part of the script but i will do more soon. i think u pasted a few things from the web. https://github.com/pfpz/SAK-MultiTool/pull/2

**Comment by PIRANY1** [comment]
I think il Replace your logging system with exit calls so it doesnt write in log files all the time

**Comment by PIRANY1** [comment]
Can you Please Accept the Pull Request?? Its here https://github.com/pfpz/SAK-MultiTool/pull/2

**Comment by PIRANY1** [comment]
Please accept the PR 😢

**Comment by PIRANY1** [comment]
Please...
